src/CollapsibleField.js

```diff
@@ -60,18 +64,54 @@ function CollapseMenu(props) {
       }
     },
     formContext = {},
+    formData = {},
     onChange,
     onAdd,
     title,
     name,
-    collapsed
+    collapsed,
+    fields,
+    propsOnChange
   } = props;
 
   const handleAdd = event => {
     event.stopPropagation();
     onAdd(event);
   };
 
+  let headerElements = [];
+  const headerItemsWrapperClass = collapsibleHeaderElements.className;
```

Comment:
expect className in all the collapsible ?

Reply:
Added default for className